packages/gcc7.rb

```diff
@@ -36,125 +24,91 @@ class Gcc7 < Package
   depends_on 'glibc'
 
   def self.build
-    # previous compile issue
-    # #{CREW_PREFIX}/bin/ld: cannot find crti.o: No such file or directory
-    # #{CREW_PREFIX}/bin/ld: cannot find /usr/lib64/libc_nonshared.a
-    ENV["LIBRARY_PATH"] = "#{CREW_LIB_PREFIX}"   # fix x86_64 issues
-    system "mkdir -p objdir"
-    Dir.chdir("objdir") do
+    ENV['LIBRARY_PATH'] = CREW_LIB_PREFIX   # fix x86_64 issues
+    system 'mkdir -p build'
```

Comment:
Since you are in a remove all system commands mood: `Dir.mkdir('build')`?

Reply:
done